src/jaspTable.cpp

```diff
@@ -1072,6 +1154,28 @@ void footnotes::convertToJSONOrdered(std::map<std::string, size_t> rowNames, std
 	mergedList	= jaspObject::VectorJson_to_ArrayJson(notesToOrderMerged);
 }
 
+Rcpp::List footnotes::toRObject() const
+{
+
+	// this is not very efficient
+	Rcpp::List	notes;
+
+	for (const auto & textRest : _data)
+		for(const auto & symbolRest : textRest.second)
+			for(const tableFields & fields : symbolRest.second)
+			{
+				Rcpp::List note = Rcpp::List::create(
+					Rcpp::Named("text")		= textRest.first,
+					Rcpp::Named("symbol")	= symbolRest.first
+//					TODO: I do not understand the data in here, or how to convert it to R...
+//					Rcpp::Named("rows")		= fields.rowsToJSON(),
```

Comment:
We can just convert to a Rcpp::List and be done with it no?
The information is available then and that should be enough for now.